disnake/types/channel.py

```diff
@@ -44,10 +44,10 @@ class PermissionOverwrite(TypedDict):
 
 class _BaseChannel(TypedDict):
     id: Snowflake
-    name: str
```

Comment:
Due to this change, the `ThreadChannel` and `GuildDirectory` types are missing a `name` field now

Reply:
https://github.com/DisnakeDev/disnake/pull/383/commits/2095bf3379d830fdf8f30f3a3a0360b2809cbb74